plugin/pkg/admission/imagepolicy/admission.go

```diff
@@ -181,6 +199,12 @@ func (a *Plugin) admitPod(pod *api.Pod, attributes admission.Attributes, review
 		return errors.New("one or more images rejected by webhook backend")
 	}
 
+	if review.Status.AttributeAnnotations == nil {
+		return nil
+	}
```

Comment:
nit: you don't need this - nil in go is equivalent to an empty slice, so the for loop will just skip.

Reply:
Thanks! Done